source/class/cv/ui/structure/pure/WgPluginInfo.js

```diff
@@ -20,6 +20,10 @@
 
 /**
  * Adds an dynamic field to the visu that shows live information from a WireGate plugin.
+ * 
+ * Note: In the web root, i.e. the directory /var/www/ must be this service
+ * script "installed":
```

Comment:
... directory /var/www/ this service must be "installed"...

Sounds better to me, or not?